packages/@romefrontend/compiler/scope/globals.ts

```diff
@@ -324,7 +324,7 @@ export const browser = [
 	"CSSRuleList",
 	"CSSStyleDeclaration",
 	"CSSStyleRule",
-	"CSSStyleSheet",
+	"CSSStylesheet",
```

Comment:
Seems to be a bad find+replace

```suggestion
	"CSSStyleSheet",
```